site/troubleshooting.md

```diff
@@ -86,7 +86,11 @@ The terms used here are explained further at
 [how it works](how-it-works.html).
 
 The 'Our name' line identifies the local weave router as a peer in the
-weave network.
+weave network. It displays both the peer name and the peer's nickname
+in parenthesis; the peer name defaults to a MAC address, whilst the
```

Comment:
We don't provide an option to override the peer name, so saying that it "defaults to" a MAC is wrong. Also "a MAC"? *what* MAC? Yes, how-it-works (which, incidentally, needs updating to mention the nickname) contains similar wording but the context there is the name *format*, i.e. "defaults to a MAC" means "the default format of a name is MAC" (I'm not exactly delighted with that phrasing either though).